webapp/src/components/tutorialValidators.tsx

```diff
@@ -81,6 +82,9 @@ export class BlocksExistValidator extends CodeValidatorBase {
         } else if (disabledBlocks.length > 0) {
             isValid = false;
             errorDescription = lf("Make sure your blocks are connected to the rest of your code like this.");
+        } else if (insufficientBlocks.length > 0) {
+            isValid = false;
+            errorDescription = lf("Make sure you have enough blocks in your workspace. It should look like this.");
```

Comment:
Since it will send this message if the block exists but is disconnected, perhaps we could rephrase slightly: "Make sure you have enough blocks connected to your workspace. It should look like this.